src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -276,11 +276,6 @@ internal static async Task<Stream> GetPythonDeploymentPackage(IEnumerable<string
             FileSystemHelpers.EnsureDirectory(packagesLocation);
 
             // Only one of the remote build or build-native-deps flag can be chosen
-            if (buildNativeDeps && buildOption == BuildOption.Remote)
-            {
-                throw new CliException("Cannot perform '--build-native-deps' along with '--build remote'");
```

Comment:
I am curious, why remove this?

Reply:
Good catch.

Reply:
Actually, this condition is not reachable since the `buildNativeDeps` flag will switch `buildOption` to `BuildOption.Container`. Basically, this is a no ops.